Makefile

```diff
@@ -8,13 +8,24 @@ BENCHTIME ?= 1x
 BENCHTIMEOUT ?= 20m
 BENCHCOUNT ?= 1
 
+podman =
+# docker --version might not contain any traces of podman in the latest
+# version, search for more output
 ifeq (,$(findstring podman,$(shell docker --version 2>/dev/null)))
+	podman = yes
+endif
+
```

Comment:
[nit] Since the comment applies to the second `ifeq findstring Podman`, the blank line should be removed.
```suggestion
```